.github/workflows/build.yml

```diff
@@ -154,7 +154,7 @@ jobs:
             export CCACHE_DIR=`pwd`/ccache
             mkdir $CCACHE_DIR
             export ARTIFACTDIR=`pwd`/buildartifacts
-            git config --global --add safe.directory /github/workspace/sources/nuttx
+            git config --global --add safe.directory /github/workspace/sources
```

Comment:
Do we know for sure if this will cover subdirectories with the actual repos? It was unclear to me in the git docs.

Reply:
Fail, so let's add 
git config --global --add safe.directory /github/workspace/sources/apps